src/Controller/SantaController.php

```diff
@@ -307,6 +307,9 @@ public function sendMessages(MessageDispatcher $messageDispatcher, Request $requ
             $messageDispatcher->dispatchRemainingMessages($secretSanta, $application);
         } catch (MessageDispatchTimeoutException $e) {
             $timeout = true;
+        } catch (MessageSendFailedException $e) {
+            $secretSanta->addError($e->getMessage(), $e->getRecipient()->getIdentifier());
+            $error = true;
```

Comment:
@damienalexandre do we need to log and notify bugsnag when a message could not be sent?

Reply:
Not sure. We do it for discord and it is always the same reason so it feel not relevant but one day... A message could fail for legit reason (like the time where is message size was an issue)